networkx/algorithms/graph_hashing.py

```diff
@@ -4,10 +4,36 @@
 For now, only Weisfeiler-Lehman hashing is implemented.
 """
 
-from collections import Counter
+from collections import Counter, defaultdict
+from copy import Error
 from hashlib import blake2b
 
-__all__ = ["weisfeiler_lehman_graph_hash"]
+__all__ = ["weisfeiler_lehman_graph_hash", "weisfeiler_lehman_subgraph_hashes"]
+
+
+def _hash_label(label, digest_size):
+    return blake2b(label.encode("ascii"), digest_size=digest_size).hexdigest()
+
+
+def _init_node_labels(G, edge_attr, node_attr):
+    if node_attr:
+        return {u: str(G.nodes[u][node_attr]) for u in G.nodes()}
+    elif edge_attr:
+        return {u: "" for u in G.nodes()}
+    else:
+        return {u: str(G.degree(u)) for u in G.nodes()}
+
+
+def _neighborhood_aggregate(G, node, node_labels, edge_attr=None):
+    """
+    Compute new labels for given node by aggregating
+    the labels of each node's neighbors.
+    """
+    label_list = [node_labels[node]]
+    for nei in G.neighbors(node):
+        prefix = "" if not edge_attr else str(G[node][nei][edge_attr])
+        label_list.append(prefix + node_labels[nei])
```

Comment:
```suggestion
    for nbr in G.neighbors(node):
        prefix = "" if edge_attr is not None else str(G[node][nbr][edge_attr])
        label_list.append(prefix + node_labels[nbr])
```
Many objects map to False in an "if" clause. So safer to check directly for the object "None"

Reply:
I think the correct version would be ` edge_attr is None` (inverted logic) but will change to check for None